test/intl402/DateTimeFormat/prototype/formatRange/argument-tonumber-throws.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2016-2019 Leonardo Balter, Igalia S.L. All rights reserved.
```

Comment:
@Ms2ger this reads as though @leobalter works at Igalia; can you move the second attribution to a different line?

Reply:
I confirm it's weird to see both names together.

I don't mind if you remove my name attribution and keep it to a single line.

Also we conventionally go with a single year for the copyright, where the range is usually for legacy tests. I believe the following should be ideal here:

```
// Copyright 2019 Igalia S.L. All rights reserved.
```